fiftyone/utils/ptv3.py

```
@@ -0,0 +1,309 @@
+            raise ValueError("No embeddings available; call predict() first")
+
+        return self._embeddings
+
+    def embed(self, arg):
+        self.predict(arg)
+        return self.get_embeddings()[0]
+
+    def embed_all(self, args):
+        return np.stack([self.embed(arg) for arg in args])
+
+    def _build_input(self, cloud):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_build_input`

(ANN202)